### DISCUSSION TITLE
❤️ Thank you, moshyfawn for helping the community

### DISCUSSION THREAD

**Discussion by bluebill1049** [react-hook-form/react-hook-form#7570] [Show and tell] [answered]
Maintaining an open-source in free time is hard and sometimes exhausting, and getting help from the community is a huge relief. I want to say thank you to @moshyfawn. He is giving a lot of help to answer questions from the community. I would like to encourage more people to appreciate his help and possibly to help each other along the way. At the end of the day, an open-source project is here to create a community to solve a particular problem and if we can all offer a tiny bit of help and appreciation. It would make everyone's life a bit easier.

<img width="335" alt="Screen Shot 2022-01-18 at 11 07 12 am" src="https://user-images.githubusercontent.com/10513364/149849766-22a7950a-c719-434d-88a1-f24eb20998c7.png">

Thank you.
BIll

**Comment by Moshyfawn** [comment, marked as answer]
As I mentioned earlier, I'm just happy to help convert people to RHF lol :D
Thx, @bluebill1049, true words in regards to everything! OSS is hard on multiple levels: mentally and financially at the very least! I wish it became easier throughout the years, bit despite all the efforts towards developing OSS maintainers support systems (sponsors, Open Collective), it's nowhere near the desired state, 'cuz it is hard do from the support prospective as well!

**Reply by bluebill1049** [reply]
I wish one day we can fund people like yourself who are helping everyone in the community as well.

**Reply by bluebill1049** [reply]
hey, @Moshyfawn could you check your Twitter? I have left you a message.
